openshift-ci/README.md

```diff
@@ -0,0 +1,125 @@
+OCS operator makes use of the [OpenShift-CI][1] platform for CI and automation of project lifecycle activities.
+The Openshift-CI configuration for ocs-operator lives in [openshift/release][2] repository.
+
+> NOTE: The commands and path given in this document are all relative to the openshift/release repository.
+
+For any help with the openshift-ci, reach out to the maintainers in `#forum-testplatform` on the CoreOS Slack.
+
+
+## Test configs
+
+The ocs-operator test configs live under `ci-operator/config/openshift/ocs-operator` openshift/release repository.
+Each branch of the ocs-operator repository has its own configuration file, and are named `openshift-ocs-operator-${branch}.yaml`.
+The config file format reference is available in [CONFIGURATION.md][4].
+
+These files are used to generate the [Prow][3] job configs which are present in `ci-operator/jobs/openshift/ocs-operator`. After each modification of the config files run
+
+```console
+$ make jobs
+```
+
+to generate/update the job files.
+
+### Test suites
+
+Currently, 3 tests are defined in the configs for ocs-operator.
+
+1. ocs-operator-ci - the general smoke test that verifies the PR. This test is required for PRs to be merged.
+2. ocs-operator-e2e-aws - the ocs-operator functional test suite. This test is required for PRs to be merged. This test is run on a new OCP cluster that is created by openshift-ci on AWS.
+3. red-hat-storage-ocs-ci-e2e-aws - the functional tests from [red-hat-storage/ocs-ci][5]. This test is run for each PR, but is not required for merge. This test is run on a new OCP cluster that is created by openshift-ci on AWS.
+
+Each of the above tests generates a job in the pre-submit job files.
+
+### Images
+
+The config files also define various images that will be generated for consumption during the tests. An images pre-submit job is generated that builds the images required for testing. An additional post-submit job is also generated that builds and promotes images to be mirrored.
```

Comment:
An **image's** pre-submit

Reply:
`images` here is the name of the generated job.

Reply:
I've changed to sentence to avoid this confusion.